lib/getFeatures.js

```diff
@@ -13,21 +13,25 @@ export async function fixedGetFeatures(options) {
 
     //get templated scenarious
     templatedScenarios = await Promise.all(templates.map(async (template) => {
-        let templateOptions = options;
+        const templateOptions = options;
         templateOptions.featurePaths = [template.path];
         try {
             let scenarios = await getFeaturesFromTemplate(templateOptions);
+            if (template.background) {
```

Comment:
not sure that this code will work
believe that it would be better
```javascript
scenarios[0] = scenarios[0].map(scenario => {
    const modifiedScenario = scenario;
    modifiedScenario.steps = template.background.concat(scenario.steps);
    return modifiedScenario
});
```

Reply:
I'm not sure what you mean 'not sure that this code will work'. I've check it. It works.